.github/workflows/node.js.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 14.x, 15.x]
```

Comment:
The newest version of Nodejs starts with 16.0 (I think it's 16.17.1?), so is it possible to add 16.x to the list of node versions? This type of testing is new to me, so please correct me if version 16 doesn't work for some reason with this

Reply:
Yes, I think only one version is needed and maybe it should be the 16 that seems to be everyone elses version too.